Sources/AsyncQueue/FIFOQueue.swift

```diff
@@ -20,8 +20,10 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-/// A queue that enables sending FIFO-ordered tasks from synchronous to asynchronous contexts
-public final class AsyncQueue: Sendable {
+/// A queue that executes asynchronous tasks enqueued from a nonisolated context in FIFO order.
```

Comment:
As before I think it would be helpful to have a code example showing how this class is intended to be used.

Reply:
I've been reflecting on this, and this queue doesn't really have an "intended" way to be used. It enables executing async tasks from a synchronous context in FIFO order, but unlike the `ActorQueue` there's no real limitations or intended usage here.